### DISCUSSION TITLE
Change ProgressBar to Indeterminate on Compressing

### DISCUSSION THREAD

**Discussion by VishnuSanal** [TeamAmaze/AmazeFileManager#2952] [Ideas]
Currently, the `ProgressBar` does not update whilst compressing, which would make the user (myself included!) feel like the compressing got stuck! Changing the progress bar to indeterminate would let the user know that the process is going on & it ain't stuck.

Update: On further testing, every option on the notification seems stuck at an initial value. So, would need to fix 'em too

My suggestions:

 - Use 'unknown/total_size' for Written & `unknown' for Time remaining. Or
 - Hide these & just use an indeterminate progress bar

**Comment by EmmanuelMess** [comment]
Why does compression not have a progress bar? Is it simply not implemented?

**Reply by VishnuSanal** [reply]
> Why does compression not have a progress bar? Is it simply not implemented?

Decryption API doesn't really post the updates. So no way to know how much is decrypted.

(copied from [here](https://t.me/AmazeFileManager/9286))

**Reply by EmmanuelMess** [reply]
This should be an issue that can be fixed with the "encrypted archives" solution.

**Reply by VishnuSanal** [reply]
> the "encrypted archives" solution.

Which issue?

**Reply by EmmanuelMess** [reply]
Created one: #2953
